src/selectedtests/task_mappings/version_limit.py

```diff
@@ -0,0 +1,36 @@
+"""VersionLimit class used to determine whether an Evergreen version is out of the desired range."""
+from typing import Optional
+from datetime import datetime
+
+from evergreen.api import Version
+
+
+class VersionLimit(object):
+    """Represents the point in time at which to start analyzing versions of an evergreen project."""
+
+    def __init__(
+        self, after_date: Optional[datetime] = None, after_version_id: Optional[str] = None
+    ):
+        """
+        Create a VersionLimit object.
+
+        :param after_date: The date at which to start analyzing versions of the repo.
```

Comment:
I think we should get some other people's opinion on the language here. I find 'after' to be confusing, especially for the version. Since we always look at versions from the current point in time and go back in history, in this context, after means the opposite of how I would think of it.

Reply:
Offline we discussed: probably something with "stop" would make sense since we are moving backwards. May be worth adding a comment as well.